docs/creating.md

```diff
@@ -75,27 +75,58 @@ their usage.
 
 In addition to
 [all the features Jinja supports](https://jinja.palletsprojects.com/en/3.1.x/templates/),
-Copier includes:
-
--   All functions and filters from
-    [jinja2-ansible-filters](https://gitlab.com/dreamer-labs/libraries/jinja2-ansible-filters/).
-
-    -   This includes the `to_nice_yaml` filter, which is used extensively in our
-        context.
-
--   `_copier_answers` includes the current answers dict, but slightly modified to make
-    it suitable to [autoupdate your project safely][the-copier-answersyml-file]:
-    -   It doesn't contain secret answers.
-    -   It doesn't contain any data that is not easy to render to JSON or YAML.
-    -   It contains special keys like `_commit` and `_src_path`, indicating how the last
-        template update was done.
--   `_copier_conf` includes a representation of the current Copier
-    <!-- prettier-ignore -->
-    [Worker][copier.main.Worker] object, also slightly modified:
-    -   It only contains JSON-serializable data.
-    -   You can serialize it with `{{ _copier_conf|to_json }}`.
-    -   ⚠️ It contains secret answers inside its `.data` key.
-    -   Modifying it doesn't alter the current rendering configuration.
-    -   It contains the current commit hash from the template in
-        `{{ _copier_conf.vcs_ref_hash }}`.
-    -   Contains Operating System-specific directory separator under `sep` key.
+Copier provides all functions and filters from
+[jinja2-ansible-filters](https://gitlab.com/dreamer-labs/libraries/jinja2-ansible-filters/).
+This includes the `to_nice_yaml` filter, which is used extensively in our context.
+
+## Variables (global)
+
+The following variables are always available in Jinja templates:
+
+### `_copier_answers`
+
+`_copier_answers` includes the current answers dict, but slightly modified to make it
+suitable to [autoupdate your project safely][the-copier-answersyml-file]:
+
+-   It doesn't contain secret answers.
+-   It doesn't contain any data that is not easy to render to JSON or YAML.
+-   It contains special keys like `_commit` and `_src_path`, indicating how the last
+    template update was done.
+
+### `_copier_conf`
+
+`_copier_conf` includes a representation of the current Copier
+[Worker][copier.main.Worker] object, also slightly modified:
+
+-   It only contains JSON-serializable data.
+-   You can serialize it with `{{ _copier_conf|to_json }}`.
+-   ⚠️ It contains secret answers inside its `.data` key.
+-   Modifying it doesn't alter the current rendering configuration.
+
+Furthermore, the following keys are added:
+
```

Comment:
I think this suggestion ties in with documenting `_copier_conf.answers_file` and `_copier_conf.src_path`:

In theory, the complete output of `dataclasses.asdict(worker)` is available in `_copier_conf`, which is why the current documentation includes a link to the [Worker spec](https://copier.readthedocs.io/en/stable/reference/main/#copier.main.Worker) and this statement exists.

I don't mind adding documentation for both, but we might need to decide if there are any other worker attributes that the documentation should point out before dropping this statement.

Reply:
Ah, you're right, I totally missed the reference to the `Worker` class documentation. It would be ideal to explicitly and accurately document all available Jinja variables and in which contexts they are available (perhaps by even generating this information), but I think it goes beyond the scope of this PR, plus I don't want this PR to block merging #1733.

Reply:
Thanks, I agree wholeheartedly. :) When I started using Copier, discovering which context variables were available when was one of the hardest parts. I just recently found out about `_copier_python` and `_folder_name` by reading the code for the `_operation` PR.

Reply:
Yes, this information isn't well documented. 🙁 I hope we'll be able to improve on this part of the docs in the future. 💪🤞